packages/core/lib/debug/printer.js

```diff
@@ -243,13 +236,23 @@ class DebugPrinter {
       this.config.logger.log(DebugUtils.formatInstruction(instructions[i]));
     }
 
-    // printout current instruction
-    this.config.logger.log(DebugUtils.formatCurrentInstruction(instruction));
+    if (!instruction || instruction.pc === undefined) {
```

Comment:
? The guard is still placed just around `formatCurrentInstruction`.  I'm confused; do you intend to make the change I suggested or not?  Again, I wasn't talking about putting it just around `formatCurrentInstruction`, but about the whole instruction-printing segment.

Reply:
Will place the guard around the entire instruction printing segment